components/dashboard/src/projects/NewProject.tsx

```diff
@@ -106,10 +106,24 @@ export default function NewProject() {
     }
 
     const reconfigure = () => {
+        //track project button clicked 'add_organisation'
+        trackProjectButton({
+            button: 'add_organisation',
```

Comment:
nit: we don't know if the intention is to add an org here. this is more generic, something like "show a github page and let me decide what to do"

Reply:
thanks, it made sense to move it to where the 'Add GitHub Orgs or Account' button is actually pushed in `/new`